.github/workflows/backmerge.yml

```diff
@@ -0,0 +1,23 @@
+name: Backmerge main -> next
+
+on:
+    release:
+        types: [released]
+
+jobs:
+    backmerge:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v3
+              with:
+                  persist-credentials: false
+
+            - name: Merge main -> next
+              uses: devmasx/merge-branch@v1.4.0
+              with:
+                  type: now
+                  from_branch: main
+                  target_branch: next
+                  github_token: ${{ secrets.GITHUB_TOKEN  }}
```

Comment:
`GH_NODE_TOKEN`

Reply:
No, it's not needed on every job, just jobs that have to perform write operations